### PR TITLE
ci: testing CI status on Rawhide [IGNORE]

### PR DESCRIPTION
**Pull request by kkaarreell**
This PR will be closed later. I have opened it to verify the CI status on Rawhide as we have spotted some issues.

### CODE REVIEW

**Review by mpeters — APPROVED** [review]

File: keylime/ca_impl_openssl.py [hunk 1]
```diff
@@ -174,6 +181,14 @@ def mk_signed_cert(
     for ext in extensions:
         cert_req = cert_req.add_extension(ext, critical=False)
 
+    crit_extensions = [
```
Comment (stefanberger) [inline keylime/ca_impl_openssl.py:184]:
```diff --git a/keylime/ca_impl_openssl.py b/keylime/ca_impl_openssl.py
index efbc2a6..455f82b 100644
--- a/keylime/ca_impl_openssl.py
+++ b/keylime/ca_impl_openssl.py
@@ -181,9 +181,13 @@ def mk_signed_cert(
     for ext in extensions:
         cert_req = cert_req.add_extension(ext, critical=False)

+    ca_pubkey = cacert.public_key()
+    if not isinstance(ca_pubkey, RSAPublicKey):
+        raise Exception("Unsupport public key in CACert")
+
     crit_extensions = [
-        x509.SubjectKeyIdentifier.from_public_key(cacert.public_key()),
-        x509.AuthorityKeyIdentifier.from_issuer_public_key(cacert.public_key()),
+        x509.SubjectKeyIdentifier.from_public_key(ca_pubkey),
+        x509.AuthorityKeyIdentifier.from_issuer_public_key(ca_pubkey),
     ]

     for ext in crit_extensions:
```

**Comment by kkaarreell** [comment]
Rawhide tests have passed. Closing this PR as the issue has been resolved.

**Pull request closed without merging**